docs/docs/documentation/extension/action.md

```diff
@@ -85,14 +85,26 @@ Extends chutney actions by adding yours:
     }
     ```
 
-# Packaging configuration
-create file chutney.actions in resources/META-INF/extension with custom actions full class name
-```
-my.custom.package.CustomAction1
-my.custom.package.CustomAction2
-```
+# Package
 
-# Examples
```

Comment:
This doc is valid if custom actions are added in chutney source code.
It will be helpfull to document also how to do if chutney is already packaged.
- create a jar that contains `/resources/META-INF/extension/chutney.actions` + the jar must be available in the running chutney classpath with `-Dloader.path`
- repackage a jar that contains chutneyserver and custom actions jars using `spring-boot-maven-plugin`